distiller/quantization/range_linear.py

```diff
@@ -570,6 +571,34 @@ def get_accum_to_output_re_quantization_params(self, output_scale, output_zero_p
         return output_scale / self.accum_scale, output_zero_point
 
 
+class FP16Wrapper(nn.Module):
+    """
+    A wrapper that replaces a module with a half precision version.
+
+    Args:
+        module (nn.Module): The module to be replaced.
+        convert_input (:obj:`bool`, optional): Specifies whether an input conversion
+            to fp16 is required for forward. Default: True.
+        return_fp32 (:obj:`bool`, optional): Specifies whether the output needs
+            to be converted back to fp32. Default: True.
+    """
+    def __init__(self, module: nn.Module, convert_input=True, return_fp32=True):
+        super(FP16Wrapper, self).__init__()
+        self.module = module.half()
```

Comment:
```suggestion
        self.wrapped_module = module.half()
```
Keeping it consistent with other wrappers.